src/apps/content-editor/src/store/navContent.js

```diff
@@ -274,8 +276,6 @@ function split(tree) {
 
   walk(tree, (tree, leaf) => {
     if (leaf.hidden) {
-      // remove leaf from tree
-      tree.splice(tree.indexOf(leaf), 1);
```

Comment:
These seems to fundamentally change how walking the tree works. I would be concerned about this introducing issues for other areas this data is used.